CHANGELOG.md

```diff
@@ -10,9 +10,11 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 - Brent crude, Western Texas Intermediate (#1478)
 - electricity cost (#1482)
 - power-to-fuel system, power-to-fuel process, power-to-ammonia system, power-to-liquid process (#1483)
+- energy storage level, energy storage content (#1486)
 
 ### Changed
 - NC/BR sector division and sector individuals; KSG sector division;  EU sectors/divisions; EU climate policy (#1462)
+- class name 'storage capacity' to 'energy storage capacity', made it subclass of 'maximum value' (#1486) 
```

Comment:
We usually only list here the classes but do not have details. I would do it like that:
```suggestion
- storage capacity -> energy storage capacity (#1486) 
```